File: Tools/VisualStudioSDK/2013.RTM/VisualStudioIntegration/Tools/Wizards/Templates/1033/VB/Guids.vb
Imports System

Class GuidList
    Private Sub New()
    End Sub

    Public Const guid%ProjectClass%PkgString As String = "%PackageGuid%"
    Public Const guid%ProjectClass%CmdSetString As String = "%CmdSetGuid%"
%ToolWindowItemStart%    Public Const guidToolWindowPersistanceString As String = "%ToolGuid%"
%ToolWindowItemEnd%%EditorStart%    Public Const guid%ProjectClass%EditorFactoryString As String = "%FactoryGuid%"
%EditorEnd%
    Public Shared ReadOnly guid%ProjectClass%CmdSet As New Guid(guid%ProjectClass%CmdSetString)
%EditorStart%    Public Shared ReadOnly guid%ProjectClass%EditorFactory As New Guid(guid%ProjectClass%EditorFactoryString)
%EditorEnd%End Class

File: Tools/VisualStudioSDK/2013.RTM/VisualStudioIntegration/Tools/Wizards/Templates/1033/CS/Guids.cs
// Guids.cs
// MUST match guids.h
using System;

namespace %ProjectNamespace%.%ProjectClass%
{
    static class GuidList
    {
        public const string guid%ProjectClass%PkgString = "%PackageGuid%";
        public const string guid%ProjectClass%CmdSetString = "%CmdSetGuid%";
%ToolWindowItemStart%        public const string guidToolWindowPersistanceString = "%ToolGuid%";
%ToolWindowItemEnd%%EditorStart%        public const string guid%ProjectClass%EditorFactoryString = "%FactoryGuid%";
%EditorEnd%
        public static readonly Guid guid%ProjectClass%CmdSet = new Guid(guid%ProjectClass%CmdSetString);
%EditorStart%        public static readonly Guid guid%ProjectClass%EditorFactory = new Guid(guid%ProjectClass%EditorFactoryString);
%EditorEnd%    };
}